turbo.json

```
@@ -45,6 +47,10 @@
             ],
             "passThroughEnv": ["GH_TOKEN", "NPM_TOKEN"]
         },
+        "start": {
+            "dependsOn": ["^compile:js", "^compile:typedefs"],
```

Running the examples doesn't actually depend on typedefs.
```suggestion
            "dependsOn": ["^compile:js"],
```